eventforwarder/forward.go

```diff
@@ -137,7 +140,8 @@ func ForwardRoomEvent(
 				}
 				eventWasForwarded = true
 
-				return ForwardEventToForwarders(ctx, enabledForwarders, status, infos, l)
+				res, warning, err = ForwardEventToForwarders(ctx, enabledForwarders, status, infos, l)
```

Comment:
is this warning variable used somewhere?

Reply:
Yes, it is passed to the `reportRPCStatus`.